src/main/java/Duke.java

```diff
@@ -1,10 +1,215 @@
+     *
+     * @param t user input task
+     */
+    public static void echoUserInput(Task t, int taskNum) {
+        System.out.println("Added: " + t.toString() + "\n You currently have " + taskNum + " tasks");
+    }
+
+    /**
+     * Adds the task 't' specified by user into the task list
+     *
+     * @param taskList An array of tasks. Max size of 100
+     * @param t        user input task
+     */
+    public static void addToTaskList(Task[] taskList, Task t, int index) {
+        taskList[index] = t;
+    }
+
+    /**
+     * Modifies a task and sets its boolean isDone to true. Prints out an acknowledgement after.
+     *
+     * @param taskList An array of tasks. Max size of 100
+     * @param i        Index of the task shown when user inputs 'list'
+     */
+    public static void completeTask(Task[] taskList, int i) {
+        boolean isInvalidListSize = (i < SMALLEST_VALID_LIST_SIZE);
+        //Navigate to the given index and change the sign
+        if (isInvalidListSize) {
+            System.out.println(LINEBAR);
+            System.out.println("No task found with the given number. \nPlease type 'list' to view your current task "
+                    + "list");
+            System.out.println(LINEBAR);
+            return;
+        }
+        i--;
+        taskList[i].markAsDone();
+        System.out.println(LINEBAR);
+        System.out.println(
+                "Bueno! The following task is marked as done: \n[" + taskList[i].getStatusIcon() + "] "
+                        + taskList[i].taskDescription);
+        System.out.println(LINEBAR);
+    }
+
+    /**
+     * This function is called upon program execution.
+     *
+     * @param args System arguments that are added to the program
+     */
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
-    }
-}
+        greetUser();
+        Scanner readUserInput = new Scanner(System.in);
+        Task[] taskList = new Task[MAX_LIST_SIZE];
+        int taskIndex = 0;
+        String userInput;
+        String identifier;
+        String taskDescription;
+        String timing;
+
+        while (true) {
+            userInput = readUserInput.nextLine();
+            identifier = userInput.split(" ")[FIRST_WORD];
+
+            switch (identifier) {
+            case "done":
+                int taskNumber = Integer.parseInt(userInput.split(" ")[TASK_NUMBER]);
+                completeTask(taskList, taskNumber);
+
+                continue;
+
+            case "bye":
+                System.out.println(LINEBAR);
+                bidGoodbye();
+                System.out.println(LINEBAR);
+                return;
```

Comment:
This can be taken out of the switch statement and implemented as a guard clause as it performs a different function from the other cases.